common/djangoapps/student/models_api.py

```diff
@@ -14,6 +14,7 @@
     ALLOWEDTOENROLL_TO_UNENROLLED as _ALLOWEDTOENROLL_TO_UNENROLLED,
     DEFAULT_TRANSITION_STATE as _DEFAULT_TRANSITION_STATE,
 )
+from student.models import UserProfile
```

Comment:
Models should be imported with an underscore *into* this file so it's known they shouldn't be imported *from* this file: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/models_api.py#L8

Reply:
It now looks like the imports in the rest of the file.